src/modules/add-site/components/create-site.tsx

```diff
@@ -50,8 +50,10 @@ export default function CreateSite( {
 	const { __ } = useI18n();
 
 	return (
-		<VStack className="w-full max-w-xl mx-auto text-black" spacing={ 6 }>
-			<Heading className="text-center text-4xl">{ __( 'Add a site' ) }</Heading>
+		<VStack className="w-full max-w-[402px] mx-auto text-black" spacing={ 6 }>
+			<Heading className="text-[32px] text-gray-900 text-center" weight={ 500 }>
+				{ __( 'Add a site 2' ) }
```

Comment:
I think the number was added for debugging. Let's remove it.

```suggestion
				{ __( 'Add a site' ) }
```